app/views/artisans/ArtisanRootView.coffee

```diff
@@ -0,0 +1,63 @@
+RootView = require 'views/core/RootView'
+
+Campaigns = require 'collections/Campaigns'
+Levels = require 'collections/Levels'
+
+module.exports = class ArtisanRootView extends RootView
+  id: 'artisan-root-view'
+  excludedCampaigns = [
+    # Misc. campaigns
+    'picoctf', 'auditions'
+
+    # Campaign-version campaigns
+    #'dungeon', 'forest', 'desert', 'mountain', 'glacier'
+
+    # Test campaigns
+    'dungeon-branching-test', 'forest-branching-test', 'desert-branching-test'
+
+    # Course-version campaigns
+    #'intro', 'course-2', 'course-3', 'course-4', 'course-5', 'course-6'
+  ]
+
+  unloadedCampaigns: 0
+  campaignLevels: {}
+  loadedLevels: {}
+  data: {}
```

Comment:
These objects will be shared across all ArtisanRootView instances. Any objects or arrays that you want to be separate between views should be created in `initialize`.